msteams-platform/resources/samesite-cookie-update.md

```diff
@@ -80,6 +80,43 @@ Android WebView is a Chrome system component that allows Android apps to display
 > * You must declare first party cookies as `SameSite=Lax` or `SameSite=Strict`, as appropriate.
 > * You must declare third party cookies as `SameSite=None; Secure`.
 
+## Third party cookies deprecation
+
+All third party cookies set in the top-level domain are blocked when that domain is embedded in an iframe. This third party cookie deprecation is starting to roll out for all major browsers, with Chrome currently at a 1% rollout and other major browsers like Firefox and Safari to follow.
+
+This deprecation impacts a common scenario in which the external app is rendered within Teams in various entry points like personal apps, channel tabs, and conversational tabs.
+
+### Pop-out authentication scenario
+
+Pop-out authentication scenarios are a common method for apps to authenticate using different identity providers like Google Auth and Facebook Auth. Here's how it works:
+
+1. The rendered iframe triggers a popup that loads the selected authentication provider sign-in page.
+1. After the user logs in, the popup redirects to the domain of the opening app, where an authentication cookie is set, and the popup closes.
+1. These cookies are used inside the embedded iframe to authenticate the user.
+
+Pop-out authentication doesn't break with third party cookie deprecation because:
+
+* Chromium-based browsers like Chrome, Microsoft Edge, and Firefox allow access to cookies that are unpartitioned, secure, and `SameSite=None`. This is true when the cookies are set in a popped-out window from the iframe to be accessible in the iframe.
+
+* The browsers that aren't Chromium-based align their cookie deprecation with Chromium-based browsers.
+
+The cookie app simulates this scenario. To use the cookie app:
+
+1. Embed the cookie app in an iframe.
+1. Use the pop-out button to pop it out as a first-party domain.
+1. Navigate to the partitioned cookie page.
+1. Select **Set Cookies Using API**.
+
+This action sets multiple cookies with a combination of secure, SameSite, and partitioned attributes. Only `SameSite=None`, secure, and unpartitioned cookies are accessible within the iframe.
+
+The following screenshot represents the cookies accessible in the embedded iframe when set from the top-level popped-out window of the iframe’s URL:
+
+:::image type="content" source="../assets/images/3p-cookies.png" alt-text="Screenshot shows you the third party cookies app in Microsoft Teams.":::
+
+### Actions required for cookies set by iframe
+
+Any cookies set by the embedded iframe must pass the partitioned attribute as false. As chromium-based browsers enable CHIPS (Cookies Having Independent Partitioned State), cookies that have a missing partitioned attribute won't be set.
```

Comment:
Consider rephrasing this CTA to use active, not passive, voice. We want to make it clear that customers need to do this in order to future-proof their app as 3P cookie deprecation rolls out.

Reply:
Updated.